arm-web/2015-08-01-preview/swagger/logicManagementClient.json

```diff
@@ -0,0 +1,1299 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "LogicManagementClient",
```

Comment:
Bit this might confuse users with the LogicApps Management client, can you use a different name?